client/src/components/mapping/InputColumn/index.tsx

```diff
@@ -68,6 +78,12 @@ const InputColumn = ({ input }: Props) => {
         inputId: input.id
       }
     });
+    const updatedGroup = attributeRules.inputGroups.find(
+      group => group.id === input.inputGroupId
+    );
+    updatedGroup!.inputs = updatedGroup!.inputs.filter(i => i.id !== input.id);
```

Comment:
I had an idea to replace this (maybe), can we use the return value of the mutation deleteInput above ? this was we retrieve directly the up-to-date attribute mapping and no need to change it manually